diag_manager/fms_diag_yaml.F90

```diff
@@ -193,6 +211,8 @@ module fms_diag_yaml_mod
 end type diagYamlObject_type
 
 type (diagYamlObject_type) :: diag_yaml  !< Obj containing the contents of the diag_table.yaml
+type (varList), save :: variable_list !< List of all the variables in the diag_table.yaml
+type (fileList), save :: file_list !< List of all files in the diag_table.yaml
```

Comment:
These will need to be updated with the type name

Reply:
done in https://github.com/NOAA-GFDL/FMS/pull/944/commits/13611692fb9996116f9d871b6a81c5b528d095fc